scripts/index.js

```diff
@@ -0,0 +1,109 @@
+
+
+const tempToColor = (temp) => { 
```

Comment:
Notice that `tempToColor` and `tempToLandscape` have similar logic. Consider how you might combine these two functions to DRY up your code.